### DISCUSSION TITLE
May 16th 2024 PowerShell/OpenSSH Community Call

### DISCUSSION THREAD

**Discussion by StevenBucher98** [PowerShell/PowerShell#21502] [Community Call]
Hosted at normal time at 9:30 am PST

Agenda:
- Conferences!
   - MSBuild Presence 
   - PSConfEu
   - Techmentor
- PSResourceGet Patch release @SydneyhSmith 
- 7.5 preview release scheduled for May, republish the dev packages for 7.4.2 @daxian-dbw 
- Docs Update @sdwheeler 
   - 7.3 retirement  
- Educational/content videos @SydneyhSmith 
- Intro Leo 
- Demo on DSC v3 Include @SteveL-MSFT 

Add any demo's in the comment of this discussion!

**Comment by HCRitter** [comment]
I would like to show my Feedback-Provider module/framework: PSClippy 
based on the awesome work from Justin Grote who made a module to for registering feedback providers easily.

**Reply by SydneyhSmith** [reply]
Great I'll slot you after Steve's demo

**Comment by ThomasNieto** [comment]
I can demo my [LockFile](https://github.com/ThomasNieto/lockfile) module that creates and validates lock files for highly available scripts where only one can be active at a time.

**Reply by SydneyhSmith** [reply]
Awesome, you are up after Christian assuming there is enough time left

**Comment by alerickson** [comment]
Sorry, thought I had added this to the agenda, but would love to talk about PSResourceGet perf updates if there's time!

**Reply by alerickson** [reply]
If not, I'll save it for next month
